configs/sparse_rcnn/sparse_rcnn_r50_fpn_mstrain_480-800_3x_coco.py

```diff
@@ -0,0 +1,23 @@
+_base_ = './sparse_rcnn_r50_fpn_1x_coco.py'
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+min_values = (480, 512, 544, 576, 608, 640, 672, 704, 736, 768, 800)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='Resize',
+        img_scale=[(1333, value) for value in min_values],
+        multiscale_mode='value',
+        keep_ratio=True),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='Pad', size_divisor=32),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels'])
+]
+
+data = dict(train=dict(pipeline=train_pipeline))
+lr_config = dict(policy='step', step=[27, 33])
```

Comment:
Will this make difference from the conventional 3x schedule, i.e., the `[28, 34, 36]` setting? We need a benchmark to verify that.

Reply:
This is calculated from the iterations of Sparse RCNN, the original schedule is different from the standard 3x.